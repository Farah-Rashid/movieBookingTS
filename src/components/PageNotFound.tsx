import React from 'react'
import { Empty } from '../styles/Empty.styles'
import img from "../Assets/EmptyState.png"

const PageNotFound = () => {
  return (
    <Empty>
      <h2>Sorry, there is no result for keyword you searched.</h2>
      <img src={img} alt="no img found" />
    </Empty>
  );
};

export default PageNotFound;