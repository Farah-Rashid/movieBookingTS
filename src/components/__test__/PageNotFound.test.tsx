import PageNotFound from '../PageNotFound'
import { render } from '@testing-library/react';


describe("testing page not found", () => {
  test('should take a snapshot', () => {
    const { container } = render(<PageNotFound />)
    expect(container).toMatchSnapshot()
  })
})
