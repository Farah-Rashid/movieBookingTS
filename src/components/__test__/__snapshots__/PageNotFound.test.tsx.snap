// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing page not found should take a snapshot 1`] = `
<div>
  <article
    class="sc-bczRLJ igZeIq"
  >
    <h2>
      Sorry, there is no result for keyword you searched.
    </h2>
    <img
      alt="no img found"
      src="EmptyState.png"
    />
  </article>
</div>
`;
